test/allocate_test.cpp

```diff
@@ -45,10 +44,7 @@ TEST(Allocate, Pipeable) {
   auto thread = threadContext.get_scheduler();
   int count = 0;
 
-  schedule(thread)
-    | then([&] { ++count; })
-    | allocate()
-    | sync_wait();
+  schedule(thread) | then([&] { ++count; }) | allocate() | sync_wait();
```

Comment:
Do we want these as on-liners?

Reply:
I mostly want automatically-maintained consistency.